javascriptv3/example_code/medical-imaging/actions/delete-datastore.js

```diff
@@ -3,38 +3,38 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-import { fileURLToPath } from "url";
+import {fileURLToPath} from "url";
 
 // snippet-start:[medical-imaging.JavaScript.datastore.deleteDatastoreV3]
-import { DeleteDatastoreCommand } from "@aws-sdk/client-medical-imaging";
-import { medicalImagingClient } from "../libs/medicalImagingClient.js";
+import {DeleteDatastoreCommand} from "@aws-sdk/client-medical-imaging";
+import {medicalImagingClient} from "../libs/medicalImagingClient.js";
 
 /**
  * @param {string} datastoreID - The ID of the data store to delete.
  */
 export const deleteDatastore = async (datastoreID = "DATASTORE_ID") => {
-  const response = await medicalImagingClient.send(
-    new DeleteDatastoreCommand({ datastoreId: datastoreID }),
-  );
-  console.log(response);
-  // {
-  //   '$metadata': {
-  //           httpStatusCode: 200,
-  //           requestId: 'f5beb409-678d-48c9-9173-9a001ee1ebb1',
-  //           extendedRequestId: undefined,
-  //           cfId: undefined,
-  //           attempts: 1,
-  //           totalRetryDelay: 0
-  //        },
-  //     datastoreId: 'xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx',
-  //     datastoreStatus: 'DELETING'
-  // }
+    const response = await medicalImagingClient.send(
+        new DeleteDatastoreCommand({datastoreId: datastoreID}),
```

Comment:
```suggestion
        new DeleteDatastoreCommand({datastoreId}),
```
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer